plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -504,6 +552,40 @@
     node: sabrewulf
     protection: yes
 
+- name: Create new VM using Cloud-Init with a username and password
+  community.general.proxmox_kvm:
+    node: sabrewulf
+    api_user: root@pam
+    api_password: secret
+    api_host: helldorado
+    name: spynal
+    ide: '{ "ide2": "local:cloudinit,format=qcow2"}'
+    ciuser: mylinuxuser
+    cipassword: supersecret
+    searchdomain: "mydomain.internal"
+    nameserver: 1.1.1.1
+    net: '{"net0":"virtio,bridge=vmbr1,tag=77"}'
+    ipconfig:
+      0:
+        ip: '192.168.1.1/24'
+        gw: '192.168.1.1'
+
+- name: Create new VM using Cloud-Init with an ssh key
+  community.general.proxmox_kvm:
+    node: sabrewulf
+    api_user: root@pam
+    api_password: secret
+    api_host: helldorado
+    name: spynal
+    ide: '{ "ide2": "local:cloudinit,format=qcow2"}'
+    sshkeys: "ssh-rsa DEADBEEF00012345"
```

Comment:
I understand a complete rsa key is a bit long for an example but an ed25519 key can both fit comfortably and be a clearer example of the required format.